scripts/us_usda/quickstats/process.py

```diff
@@ -0,0 +1,280 @@
+            print('Writing response to file', response_file)
+            json.dump(response, f, indent=2)
+
+    if 'data' not in response:
+        eprint('No api records found for county', county)
+        return {'data': []}
+
+    print('# api records for', county, '=', len(response['data']))
+    return response
+
+
+def get_data(params):
+    return requests.get(f'{API_BASE}/api_GET', params=params).json()
+
+
+def get_param_values(param):
+    params = {'key': get_usda_api_key(), 'param': param}
+    response = requests.get(f'{API_BASE}/get_param_values',
+                            params=params).json()
+    return [] if param not in response else response[param]
+
+
+'''Converts a quickstats data row to a DC CSV row.
+
+data = quickstats data row
+svs = {name: {name: ..., sv: ..., unit: ...}}
+
+returns = {variableMeasured: ..., observationAbout: ..., value: ..., unit: ...}
+'''
+
+
+def to_csv_row(data_row, svs):
+    name = data_row['short_desc']
+    if data_row['domaincat_desc'] and data_row[
+            'domaincat_desc'] != 'NOT SPECIFIED':
+        name = f"{name}%%{data_row['domaincat_desc']}"
+
+    if name not in svs:
+        eprint('SKIPPED, No SV mapped for', name)
+        return None
+
+    county_code = data_row['county_code']
+    if county_code in SKIPPED_COUNTY_CODES:
+        eprint('SKIPPED, Unsupported county code', county_code)
+        return None
+
+    value = (data_row['value'] if 'value' in data_row else
+             data_row['Value']).strip().replace(',', '')
+    if value in SKIPPED_VALUES:
+        eprint('SKIPPED, Invalid value', f"'{value}'", 'for', name)
+        return None
+    value = int(value)
+
+    observation_about = f"dcid:geoId/{data_row['state_fips_code']}{county_code}" if \
+      data_row[
+        'state_fips_code'] else 'dcid:country/USA'
+
+    sv = svs[name]
+
+    return {
+        'variableMeasured': sv['sv'],
+        'observationDate': data_row['year'],
+        'observationAbout': observation_about,
+        'value': value,
+        'unit': sv['unit'],
+    }
+
+
+def to_csv_rows(api_data, svs):
+    csv_rows = []
+
+    for data_row in api_data['data']:
+        csv_row = to_csv_row(data_row, svs)
+        if csv_row:
+            csv_rows.append(csv_row)
+
+    return csv_rows
+
+
+def load_svs():
+    svs = {}
+    with open("sv.csv", newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            svs[row['name']] = row
+    return svs
+
+
+def write_csv(out, rows):
+    writer = csv.DictWriter(out, fieldnames=CSV_COLUMNS, lineterminator='\n')
+    writer.writeheader()
+    writer.writerows(rows)
+
+
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+
+def get_all_counties():
+    svs = load_svs()
+    process_survey_data(2023, svs, "output")
+
+
+def get_multiple_years():
+    start = datetime.datetime.now()
+    print('Start', start)
+
+    svs = load_svs()
+    for year in range(2000, 2024):
```

Comment:
can change 2024 to current year: `datetime.now().year`

Reply:
Done.